config/config.toml.dist

```diff
@@ -205,3 +205,9 @@ port = 7933
 domain = "pipeline"
 createNonexistentDomain = true
 workflowExecutionRetentionPeriodInDays = 3
+
+[storage]
```

Comment:
Since this is not specific to cloud provider storage API I'd name it something like
[amazon]
defaultApiRegion

[alibaba]
defaultApiRegion

Reply:
thx, i renamed these